general/clients/installing-kodi.md

```diff
@@ -43,7 +43,7 @@ title: Installing Kodi
 The benefit of Native mode over Add-On mode is that nothing will be transcoded, currently Add-On mode doesn't support audio with more than 5.1 channels, which means no Dolby Atmos or DTS-X. Since no transcoding happens, files play instantly.
 
 1. You need to delete and re-add your libraries on your Jellyfin server in order to set the **Optional Network Path** to your share locations if you haven't already done so previously, SMB/Samba is preferred, and **must be in the Windows format** using backslashes, *even if both the client and server are using Unix-like OSes*, for example **\\\192.168.1.7\movies**, wait for the import to finish on the server.
-2. If using a Unix-like OS you must create a local user and Samba user named **guest** (with no password) on your server and allow guest only access to your shares, if using Windows this may already be done for you by default, the **guest** user is hardcoded into the plugin.
+2. If using a Unix-like OS you must create a local user and Samba user named **guest** (with no password) on your server and allow guest only access to your shares, if using Windows this may already be done for you by default. Alternatively an user password combination can be provided in the the **\\\\username:password@host\share** format.
```

Comment:
```suggestion
2. If using a Unix-like OS you must create a local user and Samba user named **guest** (with no password) on your server and allow guest only access to your shares, if using Windows this may already be done for you by default. Alternatively, a user-password combination can be provided in the the **\\\\username:password@host\share** format.
```

Reply:
Thanks for the suggestion, English is not my native language so I appreciate all help.